bin/aquifer.js

```diff
@@ -15,15 +15,19 @@
 var Aquifer = {};
 Aquifer.console = require('../lib/console.api.js')(Aquifer);
 Aquifer.init = require('../lib/init.api.js')(Aquifer);
-Aquifer.init.setup();
 
 // APIs.
-Aquifer.project = require('../lib/project.api.js')(Aquifer);
-Aquifer.build = require('../lib/build.api.js')(Aquifer);
-Aquifer.refresh = require('../lib/refresh.api.js')(Aquifer);
+Aquifer.api = {
+  project: require('../lib/project.api.js')(Aquifer),
+  build: require('../lib/build.api.js')(Aquifer),
+  refresh: require('../lib/refresh.api.js')(Aquifer)
+}
+
+// Initialize.
+Aquifer.init.setup();
 
 // Commands.
-Aquifer.commands = {
+Aquifer.command = {
```

Comment:
@mattgrill I'm thinking more from the perspective of implementation, like so:

`var x = Aquifer.command.something`
instead of
`var x = Aquifer.commands.something`

Plural makes more sense though, I like that.